android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -198,7 +199,13 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
             if (view.playerView != null) {
                 view.player = player
             } else {
-                val playerView = PlayerView(context, player)
+                // PlayerView has to be initialized with Activity context
+                val currentActivity = context.currentActivity
+                if (currentActivity == null) {
+                    Log.e(MODULE_NAME, "Cannot create a PlayerView, because no activity is attached.")
```

Comment:
I was thinking of throwing here, but because of the bridge event sending arch I suppose it could be possible to not be attached to an activity anymore, so I decided to log an error instead.